scripts/prerelease.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo "Select an option for release："
+echo
+
+select VERSION in Patch Minor Major "Specific Version"
+  do
+    echo
+    if [[ $REPLY =~ ^[1-4]$ ]]; then
+      if [[ $REPLY == 4 ]]; then
+        read -p "Enter a specific version: " -r VERSION
+        echo
+        if [[ -z $REPLY ]]; then
+          VERSION=$REPLY
+        fi
+      fi
+
+      read -p "Release $VERSION - are you sure? (y/n) " -n 1 -r
```

Comment:
I think `"Submit a PR for a bump in $VERSION version - Are you sure ..."` is clearer.